notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.    # ratio and ratioRel seems different from the BTA value..
</pre><p>How big is the difference?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
reply in the same question above